### PR TITLE
[TW-588] Doc uploading data files for monitors

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

File: src/pages/docs/monitoring-your-api/setting-up-monitor.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ contextual_links:
   - type: link
     name: "Power Mocks, Monitors, and CI/CD with Postman"
     url: "https://blog.postman.com/power-mocks-monitors-and-ci-cd-with-postman-webinar-recap-part-3/"
+  - type: link
+    name: "Looping through a Data File in the Postman Collection Runner"
```
Comment (petewitcher) [inline src/pages/docs/monitoring-your-api/setting-up-monitor.md:30]: Should Data File be lowercase?

**Review by petewitcher — APPROVED** [review]
LGTM! (With one suggestion regarding capitalization.)

File: src/pages/docs/monitoring-your-api/setting-up-monitor.md [hunk 2]
```diff
@@ -64,12 +70,26 @@ Postman offers several ways to create a new collection-based monitor:
...
+
+You can upload a data file to pass Postman sets of values to use during a monitor run. Postman will substitute the values for [variables](/docs/sending-requests/variables/) in requests or scripts in the monitored collection. By uploading a data file, you can run your tests with multiple sets of values within a single monitor run.
+
+Data files are limited to 1 MB in size and can be in CSV or JSON format. Use the following structure for your data files:
+
+* **CSV** - The first row of the file must contain the variable names, one variable per column. Each additional row is considered a data row, with values for each variable. Make sure all rows have the same number of columns and use a UNIX format line ending. Postman will iterate through the data rows, running the monitored collection once for each data row in the file.
+
+* **JSON** - The file must be formatted as an array of key-value pairs. The key corresponds to a variable name, and the value will be used for that variable when running the monitored collection.
+
+To upload a data file, use the **Select File** button to locate and open your CSV or JSON file. Postman will use the data file when running your monitor.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/monitoring-your-api/setting-up-monitor.md:89]: In the documentation, I have not seen many instances of "button" used to refer to a UI element.
Should this sentence be updated to something like the following?

"To upload a data file, select **Select File** to locate and open your CSV or JSON file."

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good to me! I added one inline question. Thanks!

File: src/pages/docs/monitoring-your-api/setting-up-monitor.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/monitoring-your-api/setting-up-monitor.md:30]: In normal usage yes, but in this case it's matching the title of the linked blog post

File: src/pages/docs/monitoring-your-api/setting-up-monitor.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/monitoring-your-api/setting-up-monitor.md:89]: It's definitely our standard not to use "button" but I personally wince at "select Select" 🤦🏻 I think you're right though and it's just best to follow the correct style 😅

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊

**Pull request merged**